src/snowflake/snowpark/dataframe_stat_functions.py

```diff
@@ -345,23 +346,11 @@ def crosstab(
                     t._1 = k
                     t._2 = v
 
-            self._dataframe._session._ast_batch.eval(repr)
-
-        if self._dataframe._session._conn.is_phase1_enabled():
-            # TODO: Logic here should be
-            # ast = self._dataframe._session._ast_batch.flush()
-            # res = self._dataframe._session._conn.ast_query(ast)
-            raise NotImplementedError(
-                "TODO: Implement corr() with EvalResult in Phase1."
-            )
-
-            # Phase 0 flushes AST and encodes it as part of the query.
-        kwargs = {}
-        _, kwargs["_dataframe_ast"] = self._dataframe._session._ast_batch.flush()
```

Comment:
Why are we not flushing the ast batch anymore here?

Reply:
this operation returns a DataFrame (not a list of rows)! It is therefore lazy.